test/intl402/NumberFormat/prototype/formatRange/argument-is-infinity-throws.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.NumberFormat-formatRange
+description: Throws a RangeError exception if start or end argument is cast to an Infinity value.
+locale: [en-US]
+features: [Intl.NumberFormat-formatRange]
+---*/
+
+const nf = new Intl.NumberFormat("en-US", {signDisplay: "exceptZero"});
+
+// 5. If x or y is a non-finite Number ..., throw a RangeError exception.
```

Comment:
Hm?  The spec does not forbid ranges involving Infinity.  It only forbids `undefined`, `NaN`, and `x > y` (where `x > y` is a more complicated operation laid out in PartitionNumberRangePattern).